src/ML-RUNNER/pair_runner.cpp

```diff
@@ -0,0 +1,1381 @@
+  int *ilist;
+  int *jlist;
+  int *numneigh, **firstneigh;
+  int *runner_num_neigh, *runner_first_neighbor, *runner_jlist, *runner_types;
+
+  // Initializes flags, which signal if energy and virial need to be tallied.
+  // In a typical MD simulation, these two properties are only required for
+  // output purposes (reporting potential energy and pressure), so their
+  // collection across all processes (=tallying) can be skipped in all timesteps
+  // where no reporting has been requested.
+  ev_init(eflag, vflag);
+
+  double **x = atom->x;
+  double **f = atom->f;
+  double *q = atom->q;
+  // Number of atoms owned by this process.
+  int nlocal = atom->nlocal;
+  // Number of ghost atoms on this process. Ghost atoms are atoms owned by another
+  // process which are inside the neighbor list cutoff for local atoms on this
+  // process. We have to calculate the force contribution from local atoms to
+  // these ghost atoms.
+  int nghost = atom->nghost;
+  int ntotal = nlocal + nghost;
+  // Total number of atoms in the simulation box. This is mpi_sum(nlocal) across
+  // all processes.
+  int natoms = static_cast<int>(atom->natoms);
+  int *type = atom->type;
+  tagint *tag = atom->tag;
+
+  // Interface variables
+  bool lperiodic;
+  double *committee_energy, *committee_force, *committee_d_energy_d_strain, *lattice;
+  double *committee_atomic_charge, *committee_hirshfeld_volume, *committee_electronegativity,
+      *committee_hardness;
+
+  lattice = new double[9];
+
+  // MPI
+  int size, rank;
+  rank = comm->me;
+  size = comm->nprocs;
+
+  if (nlocal == 0) {
+    std::cout << "Error in PairRuNNer. No local atoms on process " << rank << "." << std::endl;
+    std::cout << "Try adjusting simulation box partitioning with the "
+                 "`balance` command"
+              << std::endl;
+    std::cout << "or restart the simulation using fewer processors." << std::endl;
+    MPI_Abort(world, -1);
+  }
+
+  MPI_Barrier(world);
+  if (debug) std::cout << "Entered PairRuNNer::compute" << std::endl;
+
+  // Allocate additional per-atom arrays
+  if (atom->nmax > nmax) {
+    memory->destroy(atomic_charge);
+    memory->destroy(hirshfeld_volume);
+    memory->destroy(electronegativity);
+    memory->destroy(lagrange_charges);
+    memory->destroy(de_dq);
+    memory->destroy(screening_de_dq);
+    memory->destroy(committee_storage);
+    nmax = atom->nmax;
+
+    memory->create(atomic_charge, nmax, "pair:atomic_charge");
+    memory->create(hirshfeld_volume, nmax, "pair:hirshfeld_volume");
+    memory->create(electronegativity, nmax, "pair:electronegativity");
+    memory->create(lagrange_charges, nmax, "pair:lagrange_charges");
+    memory->create(de_dq, nmax, "pair:de_dq");
+    memory->create(screening_de_dq, nmax, "pair:screening_de_dq");
+    memory->create(committee_storage, nmax, "pair:committee_storage");
+  }
+
+  // Set additional per-atom arrays to zero
+  memset(atomic_charge, 0.0, nmax * (sizeof *atomic_charge));
```

Comment:
```suggestion
  memset(atomic_charge, 0, nmax * (sizeof *atomic_charge));
```
The second argument to memset is an int, not a double. Same applies below.

Reply:
Fixed. I switched to using integer 0 here and for the other instances in the code.